content/renderer/media/webrtc/media_stream_video_webrtc_sink.cc

```diff
@@ -249,6 +249,11 @@ MediaStreamVideoWebRtcSink::MediaStreamVideoWebRtcSink(
       refresh_interval =
           base::TimeDelta::FromMicroseconds(kLowerBoundRefreshIntervalMicros);
     }
+
+#if defined(SERVICE_OFFLOADING)
+    // For WebRTCGameStreaming, keep fps above 16 unless there are new images.
```

Comment:
For WebRTCGameStreaming -> For Service Offloading

Reply:
Can I rename this by making an additional patch?
Files you haven't mentioned seem to be modified.